pandas-stubs/core/indexes/interval.pyi

```diff
@@ -371,7 +371,7 @@ def interval_range(
     start: _TimestampLike,
     end: _TimestampLike = ...,
     periods: int | None = ...,
-    freq: str | BaseOffset | None = ...,
+    freq: str | BaseOffset | pd.Timedelta | dt.timedelta | None = ...,
```

Comment:
is `pd.Timedelta` a subclass of `datetime.timedelta`? Could simplify this union (or keep, if it is more explicit).

Reply:
Yes, it is a subclass.  We're inconsistent in the stubs.  Some places we just say `pd.Timedelta`, others we have the  union.  I like the explicit aspect to make it clear that `dt.timedelta` is accepted.